moon-store-mac-os/moon-store-mac-os/Services/BaseNetworkService.swift

```diff
@@ -51,7 +51,7 @@ class BaseNetworkService {
     func postData(for path: String,
                   with parameters: [String: Any]) async throws -> Data {
         components.path = path
-        components.queryItems = makeQueryItems(parameters: parameters)
+        components.queryItems = nil
```

Comment:
What is the reason to declare the query params as `nil` when we have `parameters` value?